Makefile.include

```diff
@@ -521,3 +521,19 @@ endif
 CFLAGS := $(patsubst -D%,,$(CFLAGS))
 CFLAGS := $(patsubst -U%,,$(CFLAGS))
 CFLAGS += -include '$(RIOTBUILD_CONFIG_HEADER_C)'
+
+# Set a custom channel if needed
+ifneq (,$(filter netdev2_ieee802154,$(USEMODULE)))
```

Comment:
NACK for several reasons:

* Boards that have both an IEEE 802.15.4 radio and a cc110x radio are not clear with this configuration
* This whole block is too examples specific to go into the build system.